src/features/auth/components/PageLayout/index.js

```diff
@@ -3,35 +3,32 @@ import PropTypes from 'prop-types';
 import { Trans } from 'react-i18next';
 import { Link } from 'react-router-dom';
 
-import ResponsiveNavigation from '../../../../packages/responsive-navigation';
-
-import { Footer, Icon, Layout, Menu, Popover } from '../../../../common/components';
+import { Layout, Navbar } from '../../../../common/components';
 
 import { AUTH_ROUTER_PATHS } from '../../constants';
 
 const PageLayout = ({ children }) => (
   <Layout>
     <Layout.Header>
-      <ResponsiveNavigation>
-        {({ activePath, hideNavigation, showNavigation, isNavigationVisible, isMobile }) =>
-          isMobile ? (
-            <Popover
-              content={<Navigation activePath={activePath} isMobile />}
-              onVisibleChange={visible => (visible ? showNavigation() : hideNavigation())}
-              title={<Icon type="close" onClick={hideNavigation} />}
-              trigger="click"
-              visible={isNavigationVisible}
-            >
-              <Icon type="bars" style={{ color: '#fff' }} />
-            </Popover>
-          ) : (
-            <Navigation activePath={activePath} />
-          )
-        }
-      </ResponsiveNavigation>
+      <Navbar>
```

Comment:
Nice & tidy client code with this Navbar! Bravo

Reply:
I was thinking navbar could be an ant-based separate package or we could invlude it in ant codebase